sdk/identity/identity/review/identity.api.md

```diff
@@ -308,7 +308,7 @@ export interface TokenCachePersistenceOptions {
 export { TokenCredential }
 
 // @public
-export interface TokenCredentialOptions extends PipelineOptions {
+export interface TokenCredentialOptions extends ServiceClientOptions {
```

Comment:
Did you consider using `CommonClientOptions`?

Reply:
Ohh thank you! I'll move to this one